### ISSUE TITLE
EM text size for Divider widget not working

### ISSUE THREAD

**Issue by vgstef** [elementor/elementor#11352]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

When choosing the size of the text in a divider widget, PX, REM and VW work, but EM doesn't work.

## Steps to reproduce

1. Add a divider widget and set it for text
2. Go to Typography and try to make the text the size of 2 EM
3. The text is invisible (probably to small)

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by udidol** [comment]
Hi @vgstef ,
Why would you want to use the EM size unit? 
This is caused because EM is relative to the parent's value, and the parent has a `font-size: 0` for technical reasons. Which parent are you looking to inherit from?

**Comment by rodolphebertozzo** [comment]
Fix this : https://github.com/elementor/elementor/pull/11430

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0

Feel free to update

Thanks!
